### DISCUSSION TITLE
What is the difference between relative variable importance (RVI) and variable importance? Can I just directly use variable importance (VIMP)?

### DISCUSSION THREAD

**Discussion by JJCCLL** [kogalur/randomForestSRC#390] [Q&A]
I am working on a dataset that needs to run random survival forest analysis. I googled and found the package randomForestSRC in the following paper: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5640485/, reference 13. We are trying to get variable importance. This paper used " The relative variable importance (RVI) of each variable can be assessed using the normalized minimal depth of the maximal subtree (which can be seen in [Figure 3](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5640485/figure/F3/)). The normalized RVI values vary from 0 (most important) to 1 (least important)." I then searched randomForestSRC vignettes and couldn't locate a function to calculate RVI. What is the difference between relative variable importance (RVI) and variable importance? Can I just directly use variable importance (VIMP)? This paper (file:///C:/Users/zxc30/Downloads/08-AOAS169-2.pdf) directly used VIMP. Thank you.

**Comment by ishwaran** [comment]
"Relative Variable Importance" (RVI) is not a standard term even though the paper you cited uses it.

What they seem to be referring to is minimal depth of a maximal subtree which measures the depth at which a variable splits in the tree.   See this [vignette](https://www.randomforestsrc.org/articles/minidep.html) for more on minimal depth.

Variable importance is a measure of importance of a variable in terms of prediction error.  See this [vignette](https://www.randomforestsrc.org/articles/rfsrc-subsample.html) for more on variable importance.
